ebpf/tracer.go

```diff
@@ -135,7 +138,12 @@ func (t *Tracer) initPerfPolling() (*bpflib.PerfMap, error) {
 					return
 				}
 				atomic.AddUint64(&t.perfReceived, 1)
-				t.state.StoreClosedConnection(decodeRawTCPConn(conn))
+				cs := decodeRawTCPConn(conn)
+				if t.shouldSkipConnection(&cs, t.determineConnectionDirection(&cs)) {
```

Comment:
Is it just a convenience to filter it early here since we filter it in `getConnections`?

Reply:
`getConnections` should probably be renamed to `getActiveConnections`. 

It is possible to centralize this logic into a single place, but filtering closed + active conns before they get into the network `state` reduces unneeded storage.